packages/server/src/drizzle/landing-page/component/component.dto.ts

```
@@ -0,0 +1,63 @@
+import { componentPage, SelectComponentPageSchema } from "../page/page";
+import {
+	componentTable,
+	ContentSchema,
+	CssSchema,
+	type CreateComponentSchema,
+} from "./component";
+
+export class ComponentTable extends DBConnection {
+	constructor(TURSO_CONNECTION_URL: string, TURSO_AUTH_TOKEN: string) {
+		super(TURSO_CONNECTION_URL, TURSO_AUTH_TOKEN);
+	}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)